### ISSUE TITLE
localise icons for left hand driving side

### ISSUE THREAD

**Issue by andrewharvey** [mapbox/directions-icons#11]
Although many of the icons in this repo are driving side independent, many are driving side dependent, and it appears they are all right side driving icons.

Is anyone interested in working on left side driving versions of the icons?

If not, would you accept a PR for this?

**Comment by AndeYashwanth** [comment]
You can just flip the image programmatically right? That's what I did.

**Comment by andrewharvey** [comment]
Nope.

For some you could, but you'd still need to keep track of which icons should be and which shouldn't. Others it won't work.

For example `uturn` you can mirror.

<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/4518a61d-fcc5-494b-adf0-61601752f9be" />

But `turn_right` should be untouched, no changes needed

<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/a51c72d5-ec11-4087-b8b5-2455a2e3e56e" />

But `roundabout_left`  if you mirror that it has you turning right, and if you don't mirror it it's got you going on the wrong side of the road.

<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/acdd8e04-b1e6-4198-898c-f7c9ec70d219" />

You could take `roundabout_right`  mirror that and make it `roundabout_left`

<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/baa85397-9d10-46c3-9309-5c13a3b74b56" />

The point is all that logic should be taken care of by the library.

> On every directions API response there is a maneuver object on each step. Within the maneuver object, there are modifier and type keys. These keys are concatenated together with a _ and prefixed with direction_ to form an icon string that maps 1:1 with the icons in this repository.

For that to hold true, I think we should generate two sets of the icons, left hand drive icons and right hand drive icons. We could build those by a script, manipulating the existing icons.

**Comment by AndeYashwanth** [comment]
Yeah you are right. There should be 4 sets of icons (light,dark) x (left,right traffic).
